installer/templates/new/dashboards/sample.html.eex

```diff
@@ -52,6 +52,7 @@
       <div class="widget-convergence"
            data-widget="Graph"
            data-source="convergence"
+           data-graph-type="line"
```

Comment:
Why change the generated default to `line` instead of area which is the default?